src/ext.js

```diff
@@ -242,11 +242,13 @@ export default function ext({ translator }) {
                   type: 'items',
                   items: {
                     fontSize: {
-                      component: 'string',
-                      type: 'string',
+                      component: 'slider',
+                      type: 'number',
                       ref: 'style.font.size',
                       translation: 'properties.fontSize',
-                      expression: 'optional',
+                      min: 0,
```

Comment:
Maybe minimum should be bigger than 0, but not really sure

Reply:
1 works fine. Don't think we need to be able to set it to 0, since you can just remove the label if you want to have no label.